src/customFileParser/unpackGeometry/InstancedMesh.ts

```diff
@@ -63,20 +60,19 @@ export default function unpackInstancedMeshes(
         matrix.multiply(rotation.makeRotationAxis(yAxis, data.rotation3.y));
         matrix.multiply(rotation.makeRotationAxis(xAxis, data.rotation3.x));
         matrix.scale(data.scale);
-
         collections[data.fileId][data.triangleOffset].addMapping(
           data.nodeId, data.treeIndex, matrix);
       }
     }
-  }
 
-  // Add collections to sector group
-  Object.keys(collections).forEach(fileId => {
-    const instancedMesh = new InstancedMesh(parseInt(fileId, 10));
-    Object.keys(collections[fileId]).forEach(triangleOffset => {
-      instancedMesh.addCollection(collections[fileId][triangleOffset]);
+    // Add collections to sector group
+    Object.keys(collections).forEach(fileId => {
+      const instancedMesh = new InstancedMesh(parseInt(fileId, 10));
```

Comment:
I'm not too happy about all the parseInt's everywhere. Shouldn't we have converted these values to numbers once we parsed them instead of having to parseInt a lot of places?

Reply:
I could do something like forEach(triangleOffsetStr => {
const triangleOffset = parseInt(triangleOffsetStr, 10);
func(triangleOffset)
}

but it seems redundant, since I'm still only using triangleOffset once. The parseInts are necessary because JavaScript object keys are always converted to strings.

Reply:
More generally, a type that is `number` should be `number` after a parser has done its job, shouldn't it? But I was wrong, it is not everywhere, only two places.

Reply:
Not quite sure what you mean. If I had something like
const myObject = {1: 'hello'}
myObject.keys() would return ['1'], not [1]. It's a JavaScript thing.

Reply:
Omg sorry, my bad. You've told me this before :D

Reply:
Fricken JavaScript. Sounds like I'm good to keep this as-is?

Reply:
Yes!